projects/plugins/social/src/js/components/connection-screen/index.js

```diff
@@ -1,28 +1,16 @@
 import { Dialog, ProductOffer, TermsOfService } from '@automattic/jetpack-components';
 import { useConnection } from '@automattic/jetpack-connection';
-import { SOCIAL_STORE_ID } from '@automattic/jetpack-publicize-components';
-import { useSelect } from '@wordpress/data';
 import { __ } from '@wordpress/i18n';
 import React from 'react';
 import background from './background.svg';
 import illustration from './illustration.png';
 import styles from './styles.module.scss';
 
 const ConnectionScreen = () => {
-	const connectProps = useSelect( select => {
-		const store = select( SOCIAL_STORE_ID );
-		return {
-			apiRoot: store.getAPIRootUrl(),
-			apiNonce: store.getAPINonce(),
-			registrationNonce: store.getRegistrationNonce(),
-		};
-	} );
-
 	const { userIsConnecting, siteIsRegistering, handleRegisterSite, registrationError } =
 		useConnection( {
 			from: 'jetpack-social',
 			redirectUri: 'admin.php?page=jetpack-social',
-			...connectProps,
 		} );
```

Comment:
Those props passed to `useConnection` are optional and it uses its own local props as default. So, we don't need to pass those.